packages/utils/src/error.ts

```diff
@@ -0,0 +1,14 @@
+let set: Set<string>;
+if (process.env.NODE_ENV !== 'production') {
+  set = new Set<string>();
+}
+
+export function error(...messages: string[]) {
+  if (process.env.NODE_ENV !== 'production') {
+    const messageKey = messages.join(' ');
+    if (!set.has(messageKey)) {
+      set.add(messageKey);
+      console.error(`Base UI: ${messageKey}`);
```

Comment:
This package is meant to be reused across all the repos, the `Base UI` prefix might need adjusting.

Reply:
This is the same as the existing `warn` util, but just calls `error` not `warn`